dojo/finding_group/redis.py

```diff
@@ -0,0 +1,256 @@
+import base64
+import json
+import logging
+import os
+from dataclasses import asdict, dataclass, field
+from datetime import datetime
+from enum import StrEnum
+from functools import lru_cache
+from typing import Self
+
+import redis
+from django.conf import settings
+from django.utils.safestring import mark_safe
+
+from dojo.models import Finding
+
+logger = logging.getLogger(__name__)
+
+DD_TEST = os.getenv("DD_TEST", "False").lower() == "true"
```

Comment:
It feels weird to have this flag/variable. Can we avoid it or at least use the same pattern as our other env variables via settings.dist.py?